autoreject/autoreject.py

```diff
@@ -185,9 +185,9 @@ def get_rejection_threshold(epochs, decim=1, random_state=None,
     ch_types : str | list of str | None
         The channel types for which to find the rejection dictionary.
         e.g., ['mag', 'grad']. If None, the rejection dictionary
-        will have keys ['mag', 'grad', 'eeg', 'eog'].
-    cv : a scikit-learn cross-validation object
-        Defaults to cv=5
+        will have keys ['mag', 'grad', 'eeg', 'eog', 'hbo', 'hbr'].
+    cv : int
+        The number of folds used. Defaults to 5.
```

Comment:
Why this change?

Reply:
Thanks for catching that. That was a rebase error by me.